worlds/oot_soh/Options.py

```diff
@@ -243,6 +243,13 @@ class ShuffleTokens(Choice):
     default = 0
 
 
+class SkullsSunSong(Toggle):
+    """
+    Night Skulltula's won't be expected in logic until Sun's Song is found.
+    """
+    display_name = "Night Skulltula's Expect Sun's Song"
```

Comment:
```suggestion
class SkullsSunSong(Toggle):
    """
    Night Skulltulas won't be in logic until Sun's Song is found.
    """
    display_name = "Night Skulltulas Expect Sun's Song"
```
Edit: go with aMannus's description, still change the display name though